specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json

```diff
@@ -1774,12 +1774,14 @@
           "$ref": "#/definitions/Entity"
         },
         {
-          "required": [
-            "isNegated"
-          ],
           "properties": {
-            "isNegated": {
-              "type": "boolean"
+            "assertions": {
+              "type": "object",
+              "$ref": "#/definitions/HealthcareAssertion"
+            },
+            "name": {
+              "description": "Preferred name for the entity. Example: 'histologically' would have a 'name' of 'histologic'.",
+              "type": "string"
```

Comment:
use enum as the set of values is a closed set

Reply:
Not sure I understand - the name isnt' an enum/set of closed values

Reply:
Ignore. Misread the swagger.